### ISSUE TITLE
Total Historical Amount

### ISSUE THREAD

**Issue by jagrosh** [Patreon/patreon-java#9]
I'm looking at the docs for [Pledge](https://docs.patreon.com/#pledge) and it looks like there is a `"total_historical_amount_cents"` field in the pledge object, but I can't find the corresponding value in the Pledge object in this library.

My goal is to be able to retrieve the total amount that has been donated from all users that have ever pledged. What would be the best way to go about this?

**Comment by nperickson** [comment]
Hey @jagrosh - Sorry for the delayed response here. We are in need of doing an overhaul of our Java lib but I do have a quick fix for this.

If you modify this line https://github.com/Patreon/patreon-java/blob/master/src/main/java/com/patreon/PatreonAPI.java#L97 to add the parameter `fields[pledge]=total_historical_amount_cents` it should resolve the issue.

**Comment by emosesPatreon** [comment]
Hi @jagrosh, with the just-released 0.4.0, this is now possible by passing `Pledge.PledgeField.TotalHistoricalAmountCents` in as an optionalField.  An example: 

```
api.fetchPageOfPledges(someCampaign, 15, null, 
    Collections.singleton(Pledge.PledgeField.TotalHistoricalAmountCents))
```
